src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/OffenceSdsPlusLookupService.kt

```diff
@@ -58,7 +58,14 @@ class OffenceSdsPlusLookupService(
                 val sentenceCalculationType = SentenceCalculationType.from(sentenceAndOffence.sentenceCalculationType)
                 val sevenYearsOrMore = sevenYearsOrMore(sentenceAndOffence)
 
-                return@filter isOffenceSdsPlus(sentenceCalculationType, sentenceAndOffence, sevenYearsOrMore, moResponseForOffence, sentenceIsAfterPcsc)
+                return@filter isOffenceSdsPlus(
```

Comment:
Maybe just rename this to show there are side effects, e.g., checkIsSDSPlusAndSetIndicators. Also` return @filter` is superfluous as it's already a boolean and the last line